examples/modkit/users_info/users_info/src/api/rest/routes.rs

```diff
@@ -56,12 +56,23 @@ pub fn register_routes(
         .description("Retrieve a paginated list of users using cursor-based pagination")
         .tag("users")
         .require_auth(&Resource::Users, &Action::Read)
-        .query_param_typed("limit", false, "Maximum number of users to return", "integer")
+        .query_param_typed(
+            "limit",
+            false,
+            "Maximum number of users to return",
+            "integer",
+        )
         .query_param("cursor", false, "Cursor for pagination")
         .handler(handlers::list_users)
-        .json_response_with_schema::<modkit_odata::Page<dto::UserDto>>(openapi, http::StatusCode::OK, "Paginated list of users")
-        .with_odata_filter_doc("OData v4 filter. Examples: `email eq 'test@example.com'`, `contains(email,'@acme.com')`")
-        .query_param("$orderby", false, "OData orderby clause. Example: 'created_at desc, id desc'")
+        .json_response_with_schema::<modkit_odata::Page<dto::UserDto>>(
+            openapi,
+            http::StatusCode::OK,
+            "Paginated list of users",
+        )
+        .with_odata_filter()
```

Comment:
I think we need some single with_odata() with parameters instead of bunch of .with_odata_XXX. But I suggest to create an issue for this future improvement.